docs/pages/dev_docs/developer_installation.rst

```diff
@@ -30,13 +30,18 @@ All contributions should include tests. To run the tests, use the command
 Linting and Formatting
 ---------------------
 
-Use `ruff <https://docs.astral.sh/ruff/>`_ for linting and `black <https://black.readthedocs.io/en/stable/>`_ for formatting.
-(At some point, we will add a pre-commit hook to enforce this.)
+Use `ruff <https://docs.astral.sh/ruff/>`_ for linting and formatting. (TODO: add something about pre-commit hook.)
```

Comment:
@dr-rodriguez Want to add something about pre-commit hook here?

Reply:
```suggestion
Use `ruff <https://docs.astral.sh/ruff/>`_ for linting and formatting.    
A pre-commit hook is provided for automatic linting and formatting with ruff. To use it, run `pip install pre-commit` and then `pre-commit install --allow-missing-config`.
```